README.md

```diff
@@ -113,10 +113,14 @@ docker pull openwisp/openwisp-websocket:latest
 
 5. When the containers are ready, you can test them out by going to the domain name that you've set for the modules.
 
-Note:
+**Note:**
```

Comment:
In the intest of not repeating ourselves, I am thinking, let's leave a `Note` here saying "If you are installing for testing, please read the notes below -- include the link to the notes below"